src/components/Greeting.js

```diff
@@ -0,0 +1,21 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchGreeting } from '../redux/action_reducers';
+
+const Greeting = () => {
+  const greeting = useSelector((state) => state.greetings);
+
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchGreeting());
+  }, [dispatch]);
+
+  return (
+    <>
+      <h2>{greeting[0]?.message}</h2>
+    </>
+  );
+};
+
+export default Greeting;
```

Comment:
- [ ] Kindly work on the Front-end to show the fetched greetings and display them on the screen. Currently, no random greeting is shown on the screen. See the screenshot below:

<img width="1735" alt="Screenshot 2023-05-20 at 2 26 14 PM" src="https://github.com/akhror-valiev/react-app-rails-backend/assets/50941074/4190237d-f1ee-40f6-9a52-1b47b34b2184">